.yarn/versions/2eafeb47.yml

```
@@ -0,0 +1,5 @@
+releases:
+  "@yarnpkg/esbuild-plugin-pnp": patch
```

```suggestion
  "@yarnpkg/esbuild-plugin-pnp": minor
```
The commit message says this is a minor release